mne/transforms.py

```diff
@@ -1521,3 +1522,155 @@ def _euler_to_quat(euler):
     quat[..., 2] = cphi * ctheta * spsi - sphi * stheta * cpsi
     quat *= mult
     return quat
+
+
+def _validate_steps(param, validate_type=None, name=None):
+    """Validate that the param only contains accepted steps."""
+    for step in param:
+        if step not in ('trans', 'rotate', 'scale', 'sdr'):
+            raise TypeError(f"``step`` must be one of ``'trans', "
+                            "'rotate', 'scale', 'sdr'`` "
+                            f"got {step} instead")
+        if validate_type is not None:
+            _validate_type(param[step], validate_type, f"{name}['{step}'']")
+
+
+@verbose
+def compute_volume_registration(moving, static, steps='warp', zooms=None,
```

Comment:
I would call moving_img and static_img to suggest it's a nibabel Image

Reply:
I had it that way at first but thought `moving` and `static` alone were clearer because it matches what dipy calls it in `affine_registration`:

https://dipy.org/documentation/1.4.1./reference/dipy.align/#affine-registration

So here I think being consistent with their naming is worth it. (Plus the name is shorter.) But I don't feel strongly so if you aren't convinced by the consistency argument I'll change it